consul/structs/structs.go

```diff
@@ -245,11 +245,12 @@ type ServiceNodes []ServiceNode
 
 // NodeService is a service provided by a node
 type NodeService struct {
-	ID      string
-	Service string
-	Tags    []string
-	Address string
-	Port    int
+	ID             string
+	Service        string
+	Tags           []string
+	Address        string
+	Port           int
+	EnableTagDrift bool
```

Comment:
We had a discussion after the thread above and this might be better called `EnableTagOverride` since the local state will adopt the tag state of the catalog. We are really drifting in the same sense that we originally imagined (which is fine), so this naming might be confusing as-is.